src/test/java/wolox/training/models/UserTest.java

```diff
@@ -92,4 +97,29 @@ public void whenFindByUsernameThatNotExist_thenReturnError() {
         Optional<User> userFound = userRepository.findByUsername("monkeys");
         assertThat(userFound.isPresent()).isFalse();
     }
+
+    @Test
+    public void whenFindAllByBirthDatesAndName_thenReturnUser() {
+        Page<User> userFound = userRepository
+                .findAllByNameIgnoreCaseContainingAndBirthdateBetween(oneTestUser.getBirthDate(),
+                        oneTestUser.getBirthDate(),
+                        oneTestUser.getName(), PageRequest.of(0, 5, Sort.by("id")));
+
+        assertThat(userFound.getTotalElements() > 0).isTrue();
+        assertFalse(userFound.get().findFirst().isEmpty());
+        assertThat(oneTestUser.equals(userFound.get().findFirst().get())).isTrue();
+    }
+
+    @Test
+    public void whenFindAllByBirthDatesAndName_thenReturnError() {
+        Page<User> userFound = userRepository
+                .findAllByNameIgnoreCaseContainingAndBirthdateBetween(oneTestUser.getBirthDate(),
+                        oneTestUser.getBirthDate(),
+                        "monkeys", PageRequest.of(0, 5, Sort.by("id")));
+
+        assertThat(userFound.getTotalElements() == 0).isTrue();
+        assertTrue(userFound.get().findFirst().isEmpty());
```

Comment:
I think this is the same assertion as the previous one 😅  am I missing something?